pkg/kwok/controllers/node_controller.go

```diff
@@ -403,15 +412,26 @@ func (c *NodeController) preprocess(ctx context.Context, node *corev1.Node) erro
 
 // playStageWorker receives the resource from the playStageChan and play the stage
 func (c *NodeController) playStageWorker(ctx context.Context) {
+	logger := log.FromContext(ctx)
+
 	for ctx.Err() == nil {
 		node := c.delayQueue.GetOrWait()
 		c.delayQueueMapping.Delete(node.Key)
-		c.playStage(ctx, node.Resource, node.Stage)
+		err := c.playStage(ctx, node.Resource, node.Stage)
+		if err != nil {
```

Comment:
Certain errors should not be retried, such as notfound, conflict, and the patch itself being misformatted

Reply:
I also have an idea that if we only retry when there are network issues, 
Can we just enable **resync** in informer such that we can re-process events (constantly) after network issues are gone? If we, we don't need retry feature at all.

Reply:
> I also have an idea that if we only retry when there are network issues,
Could we enable resync in informer such that we can re-process events (constantly) after network recovers? If so, we may don't need a retry feature.


This way is not possible, for two reasons:
- For save memory, [does not store data in memory](https://github.com/kubernetes-sigs/kwok/blob/f81396847566ec7246c23331e5235b6be8011cc2/pkg/utils/informer/informer.go#L163) in many cases
- It is not suitable for large amounts of data